app/modules/individuals/schemas.py

```diff
@@ -56,6 +56,22 @@ class Meta(BaseIndividualSchema.Meta):
         fields = BaseIndividualSchema.Meta.fields + ('names',)
 
 
+class BasicNamedIndividualSchema(ModelSchema):
+    """
+    Detailed Individual schema exposes all useful fields.
+    """
+
+    names = base_fields.Function(Individual.get_name_values)
+    id = base_fields.Function(lambda ind : ind.guid)
```

Comment:
'guid' would be a more accurate name here, but FE was already getting this under 'id' in non-bugged settings so I'm respecting the FE freeze here.

Reply:
probably a hold-over from things getting passed from EDM.   agreed - we can change later when we can coordinate with FE.

Reply:
Oh yeah, agreed on the hold-over bit